lib/api/controllers/serverController.js

```diff
@@ -153,42 +161,61 @@ class ServerController extends BaseController {
       .then(() => result);
   }
 
+  /**
+   * Manage errors to be thrown
+   *
+   * @param {...args} ...args
+   * @throws {KuzzleError}
+   */
+
+  throw(...args) {
+    const [errorName] = args;
+    const placeholders = args.slice(1) || '';
+    const error = errorCodes[this.domain].subdomains[this.subdomain].errors[errorName];
```

Comment:
You need to create a dedicated error name (+code) for a KuzzleInternalError thrown whenever a domain, subdomain or error name cannot be found. 
Once done, use it here if an error cannot be found, to prevent a crash if an error cannot be found